components/business/RegisterForm.tsx

```
@@ -0,0 +1,463 @@
+  owner_last_name: string;
+  email: string;
+  phone_number: string;
+  location: {
+    address: string;
+    postal_code: string;
+    city: string;
+    country: string;
+    latitude: number;
+    longitude: number;
+  };
+  billing_details: [{}];
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)